src/index.ts

```diff
@@ -134,7 +139,14 @@ function onError(err: AxiosError) {
   const onBackoffPromise = new Promise(resolve => {
     // Calculate time to wait with exponential backoff.
     // Formula: (2^c - 1 / 2) * 1000
-    const delay = ((Math.pow(2, config.currentRetryAttempt!) - 1) / 2) * 1000;
+    let delay: number;
+    if (config.backoffType && config.backoffType === "linear") {
```

Comment:
Guessing you will need to rebase here after a few other changes I've made, and run `npm run fix` - double quotes are usually turned into single by the formatter

Reply:
done